libraries/src/Date/Date.php

```diff
@@ -292,6 +292,7 @@ public function calendar($format, $local = false, $translate = true)
 	 *
 	 * @since   1.7.0
 	 */
+        #[\ReturnTypeWillChange]
```

Comment:
This needs to be tab indented and not space indented. 
```suggestion
	#[\ReturnTypeWillChange]
```

Reply:
FILE: /********/src/libraries/src/Date/Date.php
45s
48	----------------------------------------------------------------------
45s
49	FOUND 1 ERROR AFFECTING 1 LINE
45s
50	----------------------------------------------------------------------
45s
51	 295 | ERROR | [x] Tabs must be used to indent lines; spaces are not
45s
52	     |       |     allowed
45s
53	     |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
45s
54	----------------------------------------------------------------------
45s
55	PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
45s
56	----------------------------------------------------------------------

Reply:
> This needs to be tab indented and not space indented.

Yes, that’s the one which breaks the PHP CS check in drone.